react-task/src/components/Input.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const Input = ({ type, id, placeholder, helpText }) => {
+  return (
```

Comment:
Braces and return statement are redundant

```suggestion
const Input = ({ type, id, placeholder, helpText }) => (
```

Reply:
This comment was missed